doc/nrf/ug_thingy91_gsg.rst

```diff
@@ -455,35 +484,16 @@ You can now log in to `nRF Cloud`_ with your email and the password your chose.
 After logging in, you are directed to the dashboard view that displays your device count and service usage.
 Next, you need to activate the SIM card for the Thingy:91.
 
-Activating the iBasis SIM card
-******************************
+Connecting to nRF Cloud
+***********************
 
 .. contents::
    :local:
    :depth: 2
 
+To transmit data from your Thingy:91 to nRF Cloud, you must activate your SIM card and add the Thingy:91 to your nRF Cloud account.
 
-If you are using the iBasis Subscriber Identity Module (SIM) card that comes shipped with the Thingy:91, you need to activate it in nRF Cloud.
-
-To activate the SIM card, complete the following steps:
-
-1. Scratch off the bottom area on the back of the iBasis SIM to reveal the :term:`Personal Unblocking Key (PUK)` code.
-   The :term:`Integrated Circuit Card Identifier (ICCID)` code is the first 18 digits printed on the back.
-
-      .. figure:: /images/thingy91_sim_iccid_puk.svg
-         :alt: iBASIS SIM
-
-         iBASIS SIM
-
-#. Make a note of the ICCID and PUK codes.
-   You will need them later.
-#. Pop out the nano-SIM card.
-#. Take off the silicone cover on the Thingy:91, and put the card in the nano-SIM card holder.
-
-      .. figure:: /images/thingy91_insert_sim.svg
-         :alt: Inserting SIM
-
-         Inserting SIM
+To activate the iBasis Subscriber Identity Module (SIM) card that comes shipped with the Thingy:91, complete the following steps:
```

Comment:
```suggestion
To activate the iBasis SIM card that comes shipped with the Thingy:91 and add the Thingy:91 to nRF Cloud, complete the following steps:
```
You could also add the following note before this line to match the nRF9160 DK GS:
```
.. note::

   If you activated your iBasis SIM card before, click :guilabel:`Skip this step` in Step 4 instead of filling in the information.

   If you are using a SIM card from another provider, make sure you activate it through your network operator, then click :guilabel:`Skip this step` in Step 4 instead of filling in the information.
```